stories/ng2-component-lab/input.component.exp.ts

```diff
@@ -3,6 +3,10 @@
  */
 import {experimentOn} from '@islavi/ng2-component-lab';
 
+const valueChange = (value: any): void => {
```

Comment:
why did you change this file in search component?
If we need to fix the input please add another branch.